test-framework/common/src/main/java/io/quarkus/test/common/PathTestHelper.java

```diff
@@ -60,6 +60,10 @@ private PathTestHelper() {
     public static Path getTestClassesLocation(Class<?> testClass) {
         String classFileName = testClass.getName().replace('.', File.separatorChar) + ".class";
```

Comment:
Should the change be just here i.e replace File.separatorChar with `/`?

Reply:
cc @geoand

Reply:
That would be according to the spec of `ClassLoader.getResource`

Reply:
I had the same thought, however then I saw 1668204db93d9d7786f3396ff372728d4000bb92 which added this and made me wonder

Reply:
@beikov if you do what @machi1990 proposes, do your tests pass?

Reply:
It's not about a specific test, I just found this issue in the context of a test. With that change i.e. use `/` instead, it works. I put a break point there and evaluated the expression to set the local variable and I verified that this makes testing work.

Reply:
@beikov given that we should be using `/` instead of the OS-specific separator, do you mind fixing line 61 instead? Thanks.

Reply:
There you go